tabs/pid_tuning.js

```diff
@@ -701,11 +718,8 @@ TABS.pid_tuning.initRatesPreview = function () {
     this.keepRendering = true;
     this.model = new Model($('.rates_preview'), $('.rates_preview canvas'));
 
-    var scale = d3.scale.linear().domain([900, 2100]);
-
-    this.rollScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.pitchScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.yawScale = scale.range([Math.PI * 2, -Math.PI * 2]);
+    this.$superExpo = $('.pid_tuning input[name="show_superexpo_rates"]');
```

Comment:
I am not opposed to `currentRates`, which nicely groups all the values used for rate calculation. But sticking the jQuery element `$superExpo` as a member to the object is messy. I'd rather we had `useSuperExpo` as an additional member of `currentRates`, and evaluated it by using a jQuery element in the same way we do for the rest of the `currentRates` members around line 533. Once this is done, and once the multiplication by 100 is safe (we should probably just stick it to the end of  `checkInput`, as the code in `RateCurve` also uses the values multiplied by 100), I will look into changing `RateCurve` to use `currentRates`.